src/workers/postUpdated.ts

```diff
@@ -426,6 +463,11 @@ const fixData = async ({
         sentAnalyticsReport: privacy || !authorId,
       },
       yggdrasilId: data?.id,
+      type: data?.content_type as PostType,
+      content: data?.extra?.content,
+      contentHtml: data?.extra?.content
+        ? markdown.render(data.extra.content, { mentions: [] })
```

Comment:
Mentions do not exist?

Reply:
As it's generated they don't, but since it's a shared type I think we made it like this.
So this should do.

Reply:
Although think this will work as well:
```
markdown.render(data.extra.content)
```